compiler/passes/errorHandling.cpp

```diff
@@ -912,6 +913,24 @@ void ErrorCheckingVisitor::checkCatches(TryStmt* tryStmt) {
   }
 }
 
+static void genThrowingFnError(FnSymbol* calledFn,
```

Comment:
I might call this `isseThrowingFnError` or something along those lines to make it clearer that it's talking about issuing compilation errors.